common/djangoapps/entitlements/models.py

```diff
@@ -452,3 +452,15 @@ def __unicode__(self):
             self.support_user,
             self.reason,
         )
+
+    @classmethod
+    def get_support_reasons_list(cls):
+        """
+        Method for retrieving a serializable version of the entitlement support reasons
+
+        Returns Array of reasons
+        """
+        reasons_list = []
+        for value, label in cls.ENTITLEMENT_SUPPORT_REASONS:
+            reasons_list.append({'value': value, 'label': label})
```

Comment:
Is there a particular reason you're not using the label as a key for the value?

Reply:
hmm good point, this^ is the shape of data that the front end needs for the drop down, it seemed sensible to return it that way. I could probably change it to be label as the key, if that makes more sense for the function on the model, and then parse the dict into this shape in the react, but I'm not sure if that seems much cleaner.

Reply:
Hmm ok. I suppose I was just thinking that a key value relationship would be a form more representative of the data, but the whole point of the method is to put the data in the form you need, so I'd say this is probably fine.